shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/controllers/BridgeExecutorController.java

```diff
@@ -53,6 +62,12 @@ public class BridgeExecutorController implements Reconciler<BridgeExecutor>,
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BridgeExecutorController.class);
 
+    @ConfigProperty(name = "event-bridge.executor.deployment.timeout-seconds")
+    int executorTimeoutSeconds;
+
+    @ConfigProperty(name = "event-bridge.executor.poll-interval.seconds")
+    int executorPollIntervalSeconds;
+
```

Comment:
We might have to move these to the manager and pass them to the shard with the DTO for every resource. This is because in general the guideline is that the manager has to decide everything. But I suggest to move on with your current implementation now, let's create a ticket and move forward

Reply:
Crikey, I can understand that when the properties are _dynamic_ and vary for different instances (even `image` definition of the executor could be different when we roll-out newer versions).. but these properties are static system configuration. Surely we don't want to pass those from the `manager` to the `operator`?!?!

Reply:
Sorry I had the k8s `Deployment` in mind: that definition keeps the timeout in the spec. Your current implementation does not keep that in every CRD so you're right: it is a static config. 
Let's move forward then, in case we will need to move that parameter inside the CRD definition we will consider to move it to the DTO/manager